concrete/src/Permission/Key/AddBlockToAreaAreaKey.php

```diff
@@ -125,7 +125,6 @@ protected function getAllowedBlockTypeIDs()
      */
     protected function getAllowedBlockTypeIDsFor($operation)
     {
-        $u = new User();
```

Comment:
`$u` is defined [a few lines below](https://github.com/concrete5/concrete5/pull/8355/files#diff-415ed2b80034ddb9c2a548d1ad948a8fR136)